components/Previews/WWDPreview.js

```diff
@@ -4,42 +4,40 @@ import styles from './WWDPreview.module.css';
 
 export default function WWDPreview() {
 	return (
-		<div className={styles.wrapper}>
+		<div className={styles.Wrapper}>
 			<div className='wwdpreview--head'>
 				<Flavortext title='What We Do' />
-				<p>
+				<h2 className={styles.PageTitle}>We Develop Digital Products</h2>
+				<p className={styles.Text}>
 					Lorem ipsum dolor sit amet, consectetur adipiscing elit. Cras
 					fringilla facilisis tortor non eleifend. Ut.
 				</p>
 			</div>
-			<div className='wwdpreview--items'>
+			<div className={styles.WhatWeDoWrap}>
 				<WhatWeDo
+					className={styles.WWDItem}
+					PercentGap='-100px'
```

Comment:
I'm willing to bet it's possible to derive the pixels from the percent so that it doesn't have to be provided by the developer.